tests/unit/lms/views/api/canvas/authorize_test.py

```diff
@@ -156,47 +156,48 @@ def pyramid_request(self, pyramid_request):
 
 
 class TestOAuth2RedirectError:
-    @pytest.mark.parametrize(
-        "params,error_code",
-        [
-            ({"error": "invalid_scope"}, "canvas_invalid_scope"),
-            ({"error": "unknown_error"}, None),
-            ({"foo": "bar"}, None),
-            ({"error_description": "Something went wrong"}, None),
-        ],
-    )
-    def test_it_configures_frontend_app(self, pyramid_request, params, error_code):
-        pyramid_request.params.clear()
-        pyramid_request.params.update(params)
-        pyramid_request.lti_user = None
-
-        scopes = (
-            "url:GET|/api/v1/courses/:course_id/files",
-            "url:GET|/api/v1/files/:id/public_url",
-            "url:GET|/api/v1/courses/:id",
-            "url:GET|/api/v1/courses/:course_id/sections",
-            "url:GET|/api/v1/courses/:course_id/users/:id",
-            "url:GET|/api/v1/courses/:course_id/group_categories",
-            "url:GET|/api/v1/group_categories/:group_category_id/groups",
-            "url:GET|/api/v1/courses/:course_id/groups",
+    def test_it(self, pyramid_request):
```

Comment:
I just rewrote these tests, to make them able to handle the behaviour changes I did